### PR TITLE
updating okta docs

### PR DESCRIPTION
**Pull request by danieljordan-caci**
Please look over these docs and let me know if you have any questions. You can check out the code locally by running `yarn start` after navigating into the repo following branch pull.

### CODE REVIEW

File: docs/getting-started/okta/02-csv-import.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    admin -> assigns to admin group<br/>
+    hybrid -> assigns to BOTH office and admin groups<br/>
+    </CodeBlock>
+    </TabItem>
+</Tabs>
+
+### Double Check - Triple Check
+
+:::danger
+If the `cac_edipi` or `gsa_id` is wrong, the user will not be able to log in. Additionally, please make sure that the value in the `role` column is either `office`, `admin`, or `hyrbid` and all lowercase. Please make sure to double check these values prior to importing.
```
[the remark this answers is not in the document]
Comment (danieljordan-caci) [reply docs/getting-started/okta/02-csv-import.md]: Nice catch. Changed!

**Review by traskowskycaci — APPROVED** [review]
Looks good!

File: docs/getting-started/okta/03-okta-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+:::info
+These are some common issues that have happened when implementing Okta. If your question is not answered here, please let us know so we can investigate and determine if it needs to be added here.
+:::
+
+<table align="center">
+    <tr>
+        <td align="center"><b>I am getting an "Access Denied" error from Okta when trying to log into MilMove</b></td>
+    </tr>
+    <tr>
+        <td align="center">It is likely that you are not assigned to the application in Okta, which means they are denying you access to that application when being redirected back. Please ensure that you are assigned to the appropriate group in Okta before attemtping to sign in again.</td>
```
Comment (cameroncaci) [inline docs/getting-started/okta/03-okta-troubleshooting.md:12]: Typo: attemtping

**Review by cameroncaci — CHANGES_REQUESTED** [review]
Found one typo

**Comment by cameroncaci** [comment]
There are some spellcheck pre commit hooks we can add as well in the future

File: docs/getting-started/okta/03-okta-troubleshooting.md (hunk 2 above)
Comment (danieljordan-caci) [reply docs/getting-started/okta/03-okta-troubleshooting.md]: Corrected

**Review by cameroncaci — APPROVED** [review]

**Review by deandreJones — APPROVED** [review]
Looks good, thanks daniel

**Pull request merged**